main.go

```diff
@@ -22,8 +23,12 @@ var globalDecisionRegistry DecisionRegistry = DecisionRegistry{
 
 func runServer(cfg BouncerConfig) {
 	for _, blockListCFG := range cfg.Blocklists {
-		f := getHandlerForBlockList(blockListCFG)
+		f, err := getHandlerForBlockList(blockListCFG)
+		if err != nil {
+			log.Fatal(err)
+		}
 		http.HandleFunc(blockListCFG.Endpoint, f)
+		prometheus.MustRegister(RouteHits)
```

Comment:
not sure what happens if we register it in a loop

Reply:
# HELP blocklist_requests_total Number of calls to each blocklist
# TYPE blocklist_requests_total counter
blocklist_requests_total{route="/security/blocklist"} 3
blocklist_requests_total{route="/security/blocklist2"} 4